mrp_split/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========
+Mrp Split
+=========
+
+This module extends the functionality of Manufacturing to allow you to split
+existing Manufacturing Order (MO) in *New* or *Awaiting Raw Materials* state.
+The proposed quantity to split will ensure that the originating MO will be
+ready to produce (if possible).
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to *Manufacturing > Manufacturing Orders*.
+#. Open a Manufacturing Order in state *New* or *Awaiting Raw Materials*.
+#. Click the button *Split MO*.
+#. In the wizard, click on *Compute lines* to get a quantity recommendation to
+   split.
+#. Change the *Quantity to split* if desired and click on *Split* button at
+   the bottom of the form.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/129/9.0
+
+Known issues / Roadmap
+======================
+
+* Take into account consumable products.
+* Picking direct validation lead to error.
+* Add security group related to splitting MOs?
+* Cancelled stock moves are shown as consumed raw materials.
```

Comment:
@lreficent IMO you should not worry about cancelled stock moves. Just leave it there. Changing original field to hide cancelled move is not a correct move. Cancelled stock move information still needed on some case.